.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -55,3 +57,22 @@ curl -o vars.txt "http://localhost:8086/debug/vars"
 iostat -xd 1 30 > iostat.txt
 # Attach the `profiles.tar.gz`, `vars.txt`, and `iostat.txt` output files.
 ```
+<!--  
```

Comment:
I think this part of the text below we should render on the actual page, rather than leaving it as an html comment.  

"The following sections should be filled out by the developer who fixed the
  issue."

Reply:
sounds good to me. Will make a change

Reply:
Do we want to mention if it is regression or not?

Reply:
We already have a label `regressions`. So regressions are being tracked via this label

Reply:
removed comments